game-core/src/main/java/games/strategy/engine/history/EventChild.java

```diff
@@ -6,27 +6,27 @@
  */
 public class EventChild extends HistoryNode implements Renderable {
   private static final long serialVersionUID = 2436212909638449323L;
-  public final String m_text;
-  public final Object m_renderingData;
+  private final String text;
+  private final Object renderingData;
```

Comment:
Changed accessibility of these fields from public to private, as they aren't used outside this class.